### ISSUE TITLE
Some colors have bad contrast

### ISSUE THREAD

**Issue by poperigby** [dracula/qt5#1]
Not sure if this is supposed to look like this, but the purple text and blue highlight can create some bad contrast, and I don't think this blue color is part of the Dracula palette.
![Swappshot Tue Jan 12 17:02:45 2021](https://user-images.githubusercontent.com/20866468/104392986-2e479d80-54f8-11eb-8ab4-ea5dc83b30d5.png)

**Comment by TechieAndroid** [comment]
That's odd. Are you using my latest version?

I only used the light blue for links. And yes it is part of the pallette. Every color I set I got directly from the main github.

I do have a few little bugs to work out but I don't recognize this one. It was difficult making the theme because there's few clear connections between the colors set in qt5ct, e.g. I had to set and test colors several times to see what I was affecting until I got it right.

Anyways, keep me updated and I'll fix it as soon as I can.
